### PR TITLE
Allow cancel button on INPUT overlay to be hidden

### PR DESCRIPTION
**Pull request by 96LawDawg**
If you make "cancelButtonText" and "cancelButtonIcon" null, then the cancel button is not displayed.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: https://test.virtualtabletop.io/PR-1869/pr-test (or any other room on that server)

After merging, a backup will be available at `https://beta.virtualtabletop.io/editor/PR1869-pr-test`.

**Review by ArnoldSmith86 — COMMENTED** [review]
The rest of the code usually writes it like this: `$('#buttonInputCancel').style.display = "initial"` (including your grid code).

**Comment by 96LawDawg** [comment]
> The rest of the code usually writes it like this: `$('#buttonInputCancel').style.display = 'initial'` (including your grid code).

Code consistency is not going to be my strongpoint.  But it may be a little strong to say the "rest" of the code.  I copied this syntax from editmode.js lines 55-75.  But I like your syntax better anyway.

> Hmm, also I would expect the Go button to still be on the right. Might be enough to set `visibility` instead of `display` (needs different values though). `visibility` makes the element still take up its usual space but be invisible (and non-interactive).

I couldn't figure out how to do that.  Visibility fixes it.  Changed.

**Review by ArnoldSmith86 — CHANGES_REQUESTED** [review]
On production, the button only shows an icon if you only specify an icon. Here it shows "Cancel". Not sure why yet.

File: client/js/widgets/widget.js [hunk 1]
```diff
@@ -2330,9 +2331,10 @@ export class Widget extends StateManaged {
       if(!o.confirmButtonText && !o.confirmButtonIcon){
         confirmButtonText = "Go";
       }
-      if(!o.cancelButtonText && !o.cancelButtonIcon){
+      if (o.cancelButtonText === null && o.cancelButtonIcon === null) {
+        $('#buttonInputCancel').style.visibility = "hidden";
+      } else (!o.cancelButtonText && !o.cancelButtonIcon)
```
Comment (ArnoldSmith86) [inline client/js/widgets/widget.js:2336]: This needs to be `else if`.

**Review by ArnoldSmith86 — APPROVED** [review]
:+1:

File: client/js/widgets/widget.js (hunk 1 above)
Comment (96LawDawg) [reply client/js/widgets/widget.js]: Good catch.  I wish I knew how to code.  Thanks for putting up with me.

**Pull request merged**